pom.xml

```diff
@@ -43,8 +43,8 @@
   <properties>
     <license.title>SonarQube Cobertura Plugin</license.title>
 
-    <sonar.version>6.1</sonar.version>
-    <sonar-java.version>4.2</sonar-java.version>
+    <sonar.version>6.4</sonar.version>
```

Comment:
I understand why you put 6.4 and 6.7 in the `.tavis.yml`. However, I think, it should be better to have the same version everywhere.

Reply:
I agree. As per above I tried both 6.4 and 6.7, and both failed ... Feel free to patch it up, or I can amend. Thanks

Reply:
I applied the code localy and saw that we need an important refractor for the 6.7 version.